src/analysis/inner_product_space/gram_schmidt_ortho.lean

```diff
@@ -149,8 +173,34 @@ span_eq_span (range_subset_iff.2 $ λ i, span_mono (image_subset_range _ _) $
   gram_schmidt_mem_span _ _ le_rfl) $
   range_subset_iff.2 $ λ i, span_mono (image_subset_range _ _) $ mem_span_gram_schmidt _ _ le_rfl
 
+lemma gram_schmidt_of_orthogonal {f : ι → E} (hf : pairwise (λ i j, ⟪f i, f j⟫ = 0)) :
```

Comment:
I think we have an `is_Ortho` definition somewhere; is that only for bilinear forms? I'd somewhat expect `pairwise` in the name here.

Reply:
Can you please transfer this comment to the dependent PR #16477?